WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift

```diff
@@ -0,0 +1,273 @@
+import SwiftUI
+
+/// View to confirm the payment method before creating a Blaze campaign.
+struct BlazeConfirmPaymentView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+    @ObservedObject private var viewModel: BlazeConfirmPaymentViewModel
+
+    private let agreementText: NSAttributedString = {
+        let content = String.localizedStringWithFormat(Localization.agreement, Localization.termsOfService, Localization.adPolicy, Localization.learnMore)
+        let paragraph = NSMutableParagraphStyle()
+        paragraph.alignment = .center
+
+        let mutableAttributedText = NSMutableAttributedString(
+            string: content,
+            attributes: [.font: UIFont.caption1,
+                         .foregroundColor: UIColor.secondaryLabel,
+                         .paragraphStyle: paragraph]
+        )
+
+        mutableAttributedText.setAsLink(textToFind: Localization.termsOfService,
+                                        linkURL: Constants.termsLink)
+        mutableAttributedText.setAsLink(textToFind: Localization.adPolicy,
+                                        linkURL: Constants.adPolicyLink)
+        mutableAttributedText.setAsLink(textToFind: Localization.learnMore,
+                                        linkURL: Constants.learnMoreLink)
+        return mutableAttributedText
+    }()
+
+    init(viewModel: BlazeConfirmPaymentViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: Layout.contentPadding) {
+
+                totalAmountView
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Divider()
+
+                if !viewModel.isFetchingPaymentInfo {
+                    if viewModel.selectedPaymentMethod == nil {
+                        addPaymentMethodButton
+                            .padding(.horizontal, Layout.contentPadding)
+                    } else {
+                        cardDetailView
+                            .padding(.horizontal, Layout.contentPadding)
+                    }
+
+                } else {
+                    loadingView
+                        .padding(.horizontal, Layout.contentPadding)
+                }
+
+                Divider()
+            }
+            .padding(.vertical, Layout.contentPadding)
+        }
+        .navigationTitle(Localization.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .safeAreaInset(edge: .bottom) {
+            footerView
+        }
+        .task {
+            await viewModel.updatePaymentInfo()
+        }
+    }
+}
+
+private extension BlazeConfirmPaymentView {
+    var totalAmountView: some View {
+        VStack(alignment: .leading, spacing: Layout.contentPadding) {
+            Text(Localization.paymentTotals)
+                .bold()
```

Comment:
Should this be semibold? 
<img width="743" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/934e19f1-8425-4e4e-b326-9a61ddf9dd5e">

Reply:
Nice catch! Updated in efcd530a05064b1c588d4c7d5192e6de773c66bf.